libs/blocks/quiz/quiz.css

```diff
@@ -417,11 +409,19 @@ html[dir="rtl"] .quiz-step::after {
   .quiz-option.has-icon .quiz-option-image {
     display: flex;
   }
+
+  .quiz-step-container.top {
+    display: flex;
+  }
+  
+  .quiz-step-container.bottom{
```

Comment:
Nit: please add a space between the selector and the curly brace, to match other styles. CSS Linter might complain about this too

Reply:
I've opened another PR to resolve this. https://github.com/adobecom/milo/pull/1636

Reply:
Thanks, @overmyheadandbody, for the find, and @fullcolorcoder for the fix. It's has been merged into uar-integration